pkg/chains/formats/slsa/internal/material/material.go

```diff
@@ -66,7 +66,8 @@ func AddImageIDToMaterials(imageID string, mats *[]common.ProvenanceMaterial) er
 		if len(digest) == 2 {
 			// no point in partially populating the material
 			// do it if both conditions are valid.
-			m.URI = uriDigest[0]
+			uri := strings.TrimPrefix(uriDigest[0], "docker-pullable://")
+			m.URI = fmt.Sprintf("oci://%s", uri)
```

Comment:
Thanks @zregvart for fixing this. Can we make the prefix a constant so that we can have a single place to define what the prefixes are? Perhaps we should do it for `git+` as well.